src/main/java/Duke.java

```diff
@@ -1,103 +1,222 @@
-import java.util.Scanner;
+import javax.swing.event.DocumentEvent;
 import java.util.Arrays;
+import java.util.Scanner;
 
 public class Duke {
-    public static final String LINE_PARTITION = "    ____________________________________________________________\n";
-    public static final String FAREWELL = LINE_PARTITION +
+
+    private static final Scanner in = new Scanner(System.in);
+
+    // max no. of tasks in taskList
+    private static final int MAX_TASKS = 100;
```

Comment:
Wonderful job for keeping consistency of capital cases for all your constant values.